docs/functional_architecture.md

```diff
@@ -73,26 +73,12 @@ The same fuzzy rules that have been used to compute the ZIM path from a resource
 
 While this is expected to serve mostly for the dynamic case, we still applies them on both side (staticaly and dynamicaly) for coherency.
 
-## Content rewriting
+## Documents rewriten statically
 
-### DS rules
+For now warc2zim rewrites HTML, CSS and JS documents. For CSS and JS, this mainly consists in replacing URLs. For HTML, we also have more specific rewritting necessary (e.g. to handle base href or redirects with meta).
 
-DS (Domain Specific) rules patch javascript code with regular expressions matching. These rules are not related to url or path.
+Since 2.1, no specific domain specific (DS) rules are applied like it is done in wabac.JS because these rules are already applied in Browsertrix Crawler. For the same reason, JSON is not rewritten anymore (URL do not need to be rewritten in JSON because these URLs will be used by JS, intercepted by wombat and dynamically rewritten).
```

Comment:
```suggestion
Since 2.1, no domain specific (DS) rules are applied like it is done in wabac.JS because these rules are already applied in Browsertrix Crawler. For the same reason, JSON is not rewritten anymore (URL do not need to be rewritten in JSON because these URLs will be used by JS, intercepted by wombat and dynamically rewritten).
```